src/chrome-finder.ts

```diff
@@ -33,13 +33,13 @@ export function darwin() {
 
   execSync(
       `${LSREGISTER} -dump` +
-      ' | grep -i \'google chrome\\( canary\\)\\?.app$\'' +
+      ' | grep -i \'google chrome\\( canary\\)\\?.app.*$\'' +
```

Comment:
that's a bit aggressive (it matches ~15 lines locally for me) 

is there any narrower search we could put on it? a particular length limit perhaps or additional filtering on the JS side?

Reply:
I added a space between the .app and the wildcard to prevent the regex from returning subdirectories, that took me from 4 results down to 2. One of them is my actual correct chrome install and the other is a copy of chrome in my trash. I could limit the regex to the applications folder without too much trouble, but that could break the command for anyone who has it installed somewhere else. I'm not that great with regular expressions so I may need some help removing the "trashed" version

Reply:
@ajoy39 Just an FYI, `.*$` is redundant in most cases, because it matches everything (including nothing) up to the end of the string. It would have been better to remove `$` altogether and have `/google chrome(canary )?\.app/i` make no assumptions about what characters follow it.

I amended this as part of GoogleChrome/chrome-launcher#197, and felt I should let you know. :-)

Reply:
@Alhadis oh, thanks for letting me know! I’m not great with regex so any help is appreciated!

Reply:
@Alhadis oh, thanks for letting me know! I’m not great with regex so any help is appreciated!